website/project/metadata/utils.py

```diff
@@ -176,6 +177,8 @@ def get_options_jsonschema(options):
                         'properties': {
                             'downloads': {'type': 'integer'},
                             'version': {'type': 'integer'},
+                            'latestVersionSeen': {'type': 'string'},
```

Comment:
So apparently we never jsonschema validated registration forms via v1, so now that I've upgraded the registration process to v2 the schemas fail, these additions to file schema should fix any failures.